components/brave_wallet_ui/components/desktop/account-filter-selector/style.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Can we rename this file to account-filter-selector.style.tsx ?

Reply:
Renamed!